ROMFS/px4fmu_common/init.d-posix/px4-rc.simulator

```diff
@@ -40,11 +40,19 @@ elif [ "$PX4_SIMULATOR" = "gz" ]; then
 		echo "INFO  [init] PX4_GZ_VERBOSE not explicitly set, defaulting to: $gz_verbose."
 	fi
 
-	gz_world=$( ign topic -l | grep -m 1 -e "/world/.*/clock" | sed 's/\/world\///g; s/\/clock//g' )
-	
-	gz_version_major=$( ign gazebo --versions | sed 's/\..*//g' )
-	gz_version_minor=$( ign gazebo --versions | sed 's/'"${gz_version_major}"\.'//; s/\..*//g' )
-	gz_version_point=$( ign gazebo --versions | sed 's/'"${gz_version_major}"\.'//; s/'"${gz_version_minor}"\.'//')
+	gz_command="gz"
+	gz_sub_command="sim"
+	if [ -z $(command -v $gz_command) ];
```

Comment:
Gazebo classic has a `gz` command https://manpages.ubuntu.com/manpages/jammy/man1/gz.1.html
Therefore if you have gazebo classic and ign the script will fail as it wil believe there is gazebo garden installed

Reply:
From Jammy they are not co-installable, but I will try to find a way to fix this